search/tests/mock_search_engine.py

```diff
@@ -70,8 +70,7 @@ def value_matches(doc, field_name, field_value):
 
         if isinstance(field_value, ValueRange):
             return (
-                (field_value.lower is None or compare_value >= field_value.lower)
-                and
+                (field_value.lower is None or compare_value >= field_value.lower) and
```

Comment:
Got a pep8 violation on this line - not sure why didn't get it beforehand... but it's easy to address